### PR TITLE
BaSP-TG-64: Shared modal component

### PR DESCRIPTION
**Pull request by MaicolFreire001**
**Shared modal component**
- Created a shared modal component for success and error messages
- Changed all modals for success and error messages with the newly created

### CODE REVIEW

File: src/Components/Shared/Modal/index.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './shared.modal.module.css';
+
+const Modal = ({ type, isOpen, message, handleClose, goBack }) => {
```
Comment (Ignacio-Sormani) [inline src/Components/Shared/Modal/index.jsx:5]: This modal should be a modal that just shows an error or success message, and a button to close it. I think we shouldn't use so many conditionals

**Review by Ignacio-Sormani — CHANGES_REQUESTED** [review]

**Review by GonzaloAsencio — APPROVED** [review]

File: src/Components/Shared/Modal/shared.modal.module.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.modalOverlay {
```
[the remark this answers is not in the document]
Comment (MaicolFreire001) [reply src/Components/Shared/Modal/shared.modal.module.css:1]: Done

File: src/Components/Shared/Modal/shared.modal.module.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    margin-top: 100px;
+    text-align: center;
+}
+
+.modalHeader {
+    display: flex;
+    justify-content: space-between;
+}
+
+.modalContent h3 {
```
[the remark this answers is not in the document]
Comment (MaicolFreire001) [reply src/Components/Shared/Modal/shared.modal.module.css:25]: Done

File: src/Components/Shared/Modal/index.jsx (hunk 1 above)
Comment (MaicolFreire001) [reply src/Components/Shared/Modal/index.jsx:5]: Modified

File: src/Components/Admins/adminRow.jsx [hunk 4]
```diff
@@ -15,15 +15,14 @@ const RowAdmin = ({ listAdmin, deleteAdmin }) => {
...
-    deleteAdmin(listAdmin._id);
     await fetch(`${process.env.REACT_APP_API_URL}/admins/${listAdmin._id}`, {
       method: 'DELETE',
       headers: {
         'Content-type': 'application/json'
       },
       body: JSON.stringify(listAdmin)
     });
-    alert('The administrator was successfully removed');
+    deleteAdmin(listAdmin._id);
```
[the remark this answers is not in the document]
Comment (MaicolFreire001) [reply src/Components/Admins/adminRow.jsx:25]: Done

File: src/Components/Admins/editAdminForm.jsx [hunk 5]
```diff
@@ -11,6 +12,17 @@ function Form(props) {
     email: '',
     password: ''
   });
+  const [typeModal, setTypeModal] = useState();
```
[the remark this answers is not in the document]
Comment (MaicolFreire001) [reply src/Components/Admins/editAdminForm.jsx:15]: Done

**Review by fede230298 — APPROVED** [review]
SGMT 🚀

**Review by Ignacio-Sormani — APPROVED** [review]

**Review by JuliMonte5 — APPROVED** [review]

**Pull request merged**